styles.css

```diff
@@ -1,3 +1,1317 @@
+    background-repeat: no-repeat;
+    cursor: pointer;
+}
+.filter-clear-all {
+    background-color: transparent;
+    border: 0;
+    cursor: pointer;
+    padding-left: 12px;
+}
+
+.contdown,
+.member-sub-heading {
+    font-size: 24px;
+    font-weight: 400;
+    line-height: 30px;
+}
+
+.countdown-number {
+    font-size: 44px;
+    font-weight: 400;
+    line-height: 56px;
+}
+
+.countdown-unit {
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 28px;
+    color: #F6F6F6;
+}
+
+.details-description,
+.caption-description,
+.article-text,
+.reply-sub {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 30px;
+    color: #C2C3C5;
+}
+
+.info-category {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 26px;
+    color: #C2C3C5;
+}
+
+.info-detail {
+    font-size: 24px;
+    font-weight: 400;
+    line-height: 30px;
+}
+
+.caption-date,
+.caption-author {
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    color: #888B8E;
+}
+
+.article-tag {
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 22px;
+}
+
+.blog-details>.article-author,
+.article-date {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 30px;
+    color: #888B8E;
+}
+
+.quote {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 24px;
+    font-weight: 600;
+    line-height: 30px;
+}
+
+.rating {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 22px;
+    color: #BBBABE;
+}
+
+.blog-details>.article {
+    font-size: 30px;
+    font-weight: 400;
+    line-height: 42px;
+}
+
+.tag-item a,
+.sidebar-article p,
+.sidebar-article time,
+.tags-list li a {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: #fff;
+}
+
+.article-text {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 22px;
+    color: #888B8E;
+}
+
+.article-figure .sub-text,
+.sidebar-article>h3 {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 14px;
+    font-size: 18px;
+    font-weight: 700;
+    line-height: 28px;
+    color: #fff;
+}
+
+.categories-link,
+.filter-item {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 22px;
+    color: #C2C3C5;
+
+}
+
+h2.sidebar {
+    font-size: 24px;
+    font-weight: 400;
+    line-height: 30px;
+}
+
+.email-form {
+    background-color: #C2C3C526;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    border: none;
+
+}
+
+.catalog {
+    width: 441px;
+}
+
+.email-input {
+    background-color: transparent;
```

Comment:
The input needs text properties to style the text that will be entered.

It’s important to apply text-related CSS properties (like font-family, font-size, color) to input fields to ensure the text inside them matches the rest of your site’s design. This improves user experience by maintaining consistent typography.